packages/live-preview-sdk/src/inspectorMode/utils.ts

```diff
@@ -56,6 +61,71 @@ export function getInspectorModeAttributes(
   return null;
 }
 
+function createSelector(key: InspectorModeDataAttributes, value: string) {
+  return `[${key}="${value}"]`;
+}
+
+function hasTaggedParent(element: Element | null, sourceMap: SourceMapMetadata) {
+  const selector = [
+    createSelector(
+      sourceMap.contentful.entityType === 'Asset'
+        ? InspectorModeDataAttributes.ASSET_ID
+        : InspectorModeDataAttributes.ENTRY_ID,
+      sourceMap.contentful.entity,
+    ),
+    createSelector(InspectorModeDataAttributes.FIELD_ID, sourceMap.contentful.field),
+    createSelector(InspectorModeDataAttributes.LOCALE, sourceMap.contentful.locale),
+  ].join('');
+
+  return element?.closest(selector);
+}
+
+function isImg(node: Node): node is HTMLImageElement {
+  return node.nodeName === 'IMG';
+}
+
+/**
+ * Parses the content of the node
+ * For the `img` element use the information from `alt` otherwise use the `textContent`
+ */
+function getSourceMap(node: Node): SourceMapMetadata | null | undefined {
+  if (isImg(node)) {
```

Comment:
Thanks for adding support for images already, was planning to do this today 💯

Reply:
Just wondering, what happens for images without alt attributes? should we check if it exists first?

Reply:
If the image has no alt attribute it will return `null | undefined` and therefore the remaining logic is skipped :)

Reply:
Okay but `decode` won't complain in this case? Because for symbol we are passing an empty strings if it's undefined

Reply:
tested it: `.alt` returns always an string :)